github-sectory

```diff
@@ -94,14 +94,20 @@ class GithubSectory():
 		puts(colored.yellow("[ Getting folder content ]"))
 		self.downloadLink = parameter
 		try:
-			self.parsed = json.loads(urllib2.urlopen(self.downloadLink).read())
+			self.parsed = json.loads(urllib2.urlopen(self.downloadLink, timeout=30).read())
```

Comment:
I searched for some better alternative but could not find one. I will continue to look for the same.